### ISSUE TITLE
Patchbased Discriminative Feature Learning

### ISSUE THREAD

**Issue by wtskk** [QizeYang/PAUL#9]
Hello, as mentioned in the article, similar people, the corresponding block is similar, if one image is another image rotating 180 degrees, so the characteristics of the corresponding position can not be matched soon?

**Comment by QizeYang** [comment]
I agree, but for the person re-identification, there is basically no human body upside down.
